examples/data-centric/introduction/01-introduction-to-pygrid.ipynb

```diff
@@ -442,7 +442,7 @@
    "metadata": {},
```

Comment:
<p>If I am ok, the id of the worker is "Bob" (uppercase)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PyGrid/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>